<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>484</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Slider和Color</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Alpha</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSlider" name="SliderAlpha">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSlider" name="SliderBlue">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Green</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Red</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="SliderGreen">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Blue</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSlider" name="SliderRed">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>水平</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Dial和LCDNUmber</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QDial" name="dial">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="LCDDisplay">
           <property name="digitCount">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>LCD显示进制</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioBtnDec">
              <property name="text">
               <string>十进制</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnBin">
              <property name="text">
               <string>二进制</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnOct">
              <property name="text">
               <string>八进制</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnHex">
              <property name="text">
               <string>十六进制</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSlider" name="verticalSlider">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="verticalScrollBar">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar_2">
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>71</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeBtn">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>70</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
